packages/teams-js/src/internal/messageObjects.ts

```diff
@@ -1,9 +1,57 @@
+import { generateGUID, validateUuid } from './utils';
+
+/**
+ * @internal
+ * Limited to Microsoft-internal use
+ *
+ * MessageUUID object that holds the uuid used for processing MessageRequests and MessageResponses
+ */
+export class MessageUUID {
+  private uuid: string;
```

Comment:
Could you instead declare this in the constructor like this:

```
public constructor(private readonly uuid: string = generateGUID()) {
    validateUuid(uuid);
}
```

Reply:
Aah yes, that is much cleaner. I'll do that :)